runtime/autoload/health/provider.vim

```diff
@@ -120,6 +126,13 @@ endfunction
 function! s:check_clipboard() abort
   call health#report_start('Clipboard (optional)')
 
+  if !empty($TMUX) && executable('tmux') && executable('pbcopy') && !s:cmd_ok('pbcopy')
+    let tmux_version = matchstr(system('tmux -V'), '\d\+\.\d\+')
+    call health#report_error('pbcopy does not work with tmux version: '.tmux_version,
+          \ ['Install tmux 2.6+',
+          \  'See https://superuser.com/q/231130'])
```

Comment:
Maybe we could add...

_or use tmux together with reattach-to-user-namespace: https://github.com/ChrisJohnsen/tmux-MacOSX-pasteboard_

...which is how people used the pasteboard before tmux 2.6.